### DISCUSSION TITLE
Is it possible to have an element be a reference for multiple floating elements?

### DISCUSSION THREAD

**Discussion by stormwarning** [floating-ui/floating-ui#1770] [Q&A]
For example: we have a button with just an icon, so we have a Tooltip labelling the button on hover/focus, but this button also triggers a Menu.

I have a [basic demo](https://codesandbox.io/s/quirky-danilo-4cflnw?file=/src/DropdownMenu.tsx) showing the issue, adapted from the documentation examples that's pretty similar to my particular case.

I tried wrapping the Tooltip in `forwardRef` and merging that ref with the reference from `useFloating()` but it still doesn't seem to work. Am I missing something?

**Comment by atomiks** [comment]
I haven't properly looked at the example but yes it's possible as you can see here: https://codesandbox.io/s/old-wind-f50mzf?file=/src/App.js
